### PR TITLE
[DOC] Mention classification type in plot_haxby_glm_decoding.py

### PR DESCRIPTION
**Pull request by TamerGezici**
Addresses issue: https://github.com/nilearn/nilearn/issues/4109

### CODE REVIEW

**Review by bthirion — COMMENTED** [review]
Thx. Can you make sure that the length of the lines is no more than 80 characters ?
Best,

File: examples/02_decoding/plot_haxby_glm_decoding.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 =========================================================
 
 Full step-by-step example of fitting a :term:`GLM`
-to perform a decoding experiment.
+to perform a decoding experiment. In this decoding analysis, we will be doing a one-vs-all classification.
```
Comment (ymzayek) [inline examples/02_decoding/plot_haxby_glm_decoding.py:6]:
```suggestion
to perform a decoding experiment. In this decoding analysis,
we will be doing a one-vs-all classification.
```

File: examples/02_decoding/plot_haxby_glm_decoding.py [hunk 2]
```diff
@@ -176,7 +176,7 @@
 )
 decoder.fit(z_maps, conditions_label, groups=session_label)
 
-# Return the corresponding mean prediction accuracy compared to chance
+# Return the corresponding mean prediction accuracy compared to chance for classifying one-vs-all items.
```
Comment (ymzayek) [inline examples/02_decoding/plot_haxby_glm_decoding.py:179]:
```suggestion
# Return the corresponding mean prediction accuracy compared to chance
# for classifying one-vs-all items.
```

**Comment by ymzayek** [comment]
@TamerGezici are you able to finish this or would you like us to take over?

**Comment by TamerGezici** [comment]
I think it's good now

I have a question though, here, all-vs-one clasification is done for which "one"? Or is it all classified against one, and then averaged for each classification?

It may be worth adding that to the end of DOC if that is so.

**Comment by ymzayek** [comment]
@TamerGezici it is rather the latter, each condition classified against all for each group so you end up with `np.shape(list(decoder.cv_scores_.values()))` of `(8, 12)` and the averaging is done in L183 across all groups and conditions.

File: doc/changes/latest.rst [hunk 3]
```diff
@@ -28,6 +28,7 @@ Enhancements
 - :bdg-primary:`Code` Add ``two_sided`` option for :class:`~nilearn.image.binarize_img` (:gh:`4121` by `Steven Meisler`_).
 - :bdg-dark:`Code` :meth:`~maskers.NiftiLabelsMasker.generate_report` now uses appropriate cut coordinates when functional image is provided (:gh:`4099` by `Yasmin Mzayek`_ and `Nicolas Gensollen`_).
 - :bdg-info:`Plotting` When plotting thresholded statistical maps with a colorbar, the threshold value(s) will now be displayed as tick labels on the colorbar (:gh:`#2833` by `Nicolas Gensollen`_).
+- :bdg-primary:`Doc`  Mention the classification type (all-vs-one) in  :func:`~plot_haxby_glm_decoding` (:gh:`4122` by `TamerGezici`_).
```
Comment (ymzayek) [inline doc/changes/latest.rst:31]:
```suggestion
- :bdg-primary:`Doc`  Mention the classification type (all-vs-one) in  :ref:`sphx_glr_auto_examples_02_decoding_plot_haxby_glm_decoding.py` (:gh:`4122` by `TamerGezici`_). 
```

**Comment by ymzayek** [comment]
You also need to add your name to CITATION.cff as a first time contributor

**Comment by TamerGezici** [comment]
Done. I didn't know a simple commit like this would qualify for that. Haha!

**Comment by Remi-Gau** [comment]
> Done. I didn't know a simple commit like this would qualify for that. Haha!

You help clarifying something that was unclear to you and probably others: defo a contribution.

**Review by bthirion — APPROVED** [review]
Thx !

**Pull request merged**